src/linear_algebra/basic.lean

```diff
@@ -1835,6 +1835,28 @@ def of_submodules (p : submodule R M) (q : submodule R M₂) (h : p.map ↑e = q
 @[simp] lemma of_submodules_symm_apply {p : submodule R M} {q : submodule R M₂}
   (h : p.map ↑e = q) (x : q) : ↑((e.of_submodules p q h).symm x) = e.symm x := rfl
 
+/-- Given `f : M₁ ≃ₗ[R] M₂` and `U` a submodule of `M₂`, `f.comap U` is the
+induced `linear_equiv` from `U.comap f.to_linear_map` to `U`. -/
```

Comment:
Using the docstring of `of_submodule` as a template:
```suggestion
/-- A linear equivalence of two modules restricts to a linear equivalence from the preimage of any submodule
of the range to that submodule.

This is `linear_equiv.of_submodule` but with `comap` on the left instead of `map` on the right. --/
```

Feel free to edit the docstring of `of_submodule` if you think you can improve that one too, but it would be good for them to be consistent.

That might need line-wrapping too...